File: Data Structures & Algorithms/two-integer-sum/submission-1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        mp = dict()
        for i in range(len(nums)):
            if target - nums[i] in mp:
                return [mp.get(target - nums[i]), i]
            
            mp[nums[i]] = i;
        
        return []

File: Data Structures & Algorithms/kth-largest-element-in-an-array/submission-1.java
class Solution {
    public int findKthLargest(int[] nums, int k) {
        PriorityQueue<Integer> pq = new PriorityQueue<Integer> ((a, b) -> (b - a));
        for (int i : nums){
            pq.offer(i);
        }

        while (k > 1){
            pq.poll();
            k--;
        }

        return pq.peek();
    }
}
